nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -170,8 +178,49 @@ public void shouldFailWhenRetrievingDocumentBlobWithoutColdStorageContent() {
         }
     }
 
-    protected DocumentModel createDocument(boolean addBlobContent) {
-        DocumentModel documentModel = session.createDocumentModel("/", "anyFile", "File");
+    @Test
+    public void shouldCheckAvailabilityOfColdStorageContent() {
+        List<String> documents = Arrays.asList( //
+                moveAndRetrieveColdStorageContent(DEFAULT_DOC_NAME).getId(),
+                moveAndRetrieveColdStorageContent("anyFile2").getId(), //
+                moveAndRetrieveColdStorageContent("anyFile3").getId());
+        transactionalFeature.nextTransaction();
```

Comment:
not strictly needed

Reply:
you mean `transactionalFeature.nextTransaction();` ? if it is the case. unfortunately this is required otherwise the test fail because 
```
ColdStorageHelper.checkAvailabilityOfColdStorageContent(
                    session);
```
will perform a query and we should commit the tx.

Reply:
Yes, OK :+1: